source/gui/settingsDialogs.py

```diff
@@ -1614,6 +1614,9 @@ def makeSettings(self, settingsSizer):
 		self.reportNormalizedForCharacterNavigationCheckBox.SetValue(
 			config.conf["speech"]["reportNormalizedForCharacterNavigation"]
 		)
```

Comment:
@CyrilleB79 - I just realised, should this have been removed? Why wasn't this working?

Reply:
@seanbudd, it was (and it is) working. These lines control the checkbox's value, i.e. checked vs unchecked.

On the other hand, the modification of this PR instead deal with the control's state, i.e. enabled (normally activable) / disabled (greyed out, not activable).

These are two separate thing.